src/passes/Asyncify.cpp

```diff
@@ -1536,8 +1536,7 @@ struct Asyncify : public Pass {
     String::Split listedImports(stateChangingImports, ",");
     // TODO: consider renaming asyncify-ignore-indirect to
     //       asyncify-ignore-nondirect, but that could break users.
-    auto ignoreNonDirect =
-      options.getArgumentOrDefault("asyncify-ignore-indirect", "") == "";
```

Comment:
This line seems wrong... aren't we setting ignoreNonDirect in cases where asyncify-ignore-indirect is not set?

Reply:
Good point, this does look wrong... We do have tests for it, and this PR is flipping the value, so we should see an effect from that which could help explain what's going on.